### PR TITLE
`Lectures`: Fix an issue with a readonly validation for attachments

### PR DESCRIPTION
**Pull request by az108**
### Motivation and Context
Issue was fixed where, during the Creation of Lecture Units the version tab would show object object instead of a value.
https://github.com/ls1intum/Artemis/issues/8439

### Description
I added the intended value to version and removed the readonly as it was redundant. The readonly is considered in the html file already and the value 1 is still readonly after the changes.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to Lecture Unit
4. Then Create

#### Performance Review
- [ ] I (as a reviewer) confirm that the client changes (in particular related to REST calls and UI responsiveness) are implemented with a very good performance 
#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

## Summary by CodeRabbit

- **Refactor**
	- Simplified the data structure and interactions related to grading criteria in exercises by removing the handling of `pointsPerCriterion` across various components and services. This change streamlines the process of managing exercise scores and results, enhancing the user experience by focusing on essential data.

### CODE REVIEW

**Review by sarpsahinalp — APPROVED** [review]
Tested on ts2 works as expected!

**Review by Jan-Thurner — DISMISSED** [review]
code lgtm

**Comment by iyannsch** [comment]
Works as expected, the `object object` is now a `1` - good job there :) 

One small feedback aspect though, it took me 15 mins just to find the part you changed in the Artemis Lecture/Unit Management system. In the course, I created a text-based Unit, which is obviously not affected by your changes nor the initial bug.
In future, I´d love to see you write a bit more detail in the "Reviewer Nudging" to help me recreate the fault. I was just able to detect it now, as the issue's title contains `pdf` 👀😂

Thanks for your work!

**Review by iyannsch — APPROVED** [review]

**Comment by az108** [comment]
I will keep that in mind thanks!

File: src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export class AttachmentUnitFormComponent implements OnInit, OnChanges {
             name: [undefined as string | undefined, [Validators.required, Validators.maxLength(255)]],
             description: [undefined as string | undefined, [Validators.maxLength(1000)]],
             releaseDate: [undefined as dayjs.Dayjs | undefined],
-            version: [{ value: 1, disabled: true, readOnly: true }],
+            version: [1],
```
Comment (maximiliansoelch) [inline src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts:91]: The field should still be disabled! Also see https://github.com/ls1intum/Artemis/pull/8114/files#r1510001615

```suggestion
            version: [{ value: 1, disabled: true }],
```

**Review by maximiliansoelch — CHANGES_REQUESTED** [review]

**Comment by az108** [comment]
works after the commit change aswell. Tested it locally

**Review by maximiliansoelch — APPROVED** [review]
Thanks for implementing the suggestion, the linked issue is solved 👍

**Review by Jan-Thurner — APPROVED** [review]

**Review by iyannsch — APPROVED** [review]

**Review by egekurt123 — APPROVED** [review]
LGTM

**Review by sarpsahinalp — APPROVED** [review]
Reapprove

**Comment by az108** [comment]
> Thanks for implementing the suggestion, the linked issue is solved 👍

Thanks for suggesting :)

**Pull request merged**